assembly/assembly-main/src/assembly/bin/launcher.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+
+# See: https://sipb.mit.edu/doc/safe-shell/
+set -e
+trap exit SIGHUP SIGINT SIGTERM
+
+init_global_variables () {
+
+  # For coloring console output
+  BLUE='\033[1;34m'
+  GREEN='\033[0;32m'
+  NC='\033[0m'
+
+  ### Define various error and usage messages
+  USAGE="
+Usage:
+  `basename "$0"` [COMMAND]
+     start                              Starts Che server in new console
```

Comment:
Container is started as detached docker container. I'm not sure it is precise and meaningful to say that it is launched in a new console